packages/destination-actions/src/destinations/customerio/index.ts

```diff
@@ -67,7 +65,7 @@ const destination: DestinationDefinition<Settings> = {
     },
     {
       name: 'Create or Update Device',
-      subscribe: 'type = "track" and event = "Application Installed"',
+      subscribe: 'type = "track" and ( event = "Application Installed" or event = "Application Opened" )',
```

Comment:
Same here – you can omit the `type = "track"` to avoid grouping since it is not yet supported.

```suggestion
      subscribe: 'event = "Application Installed" or event = "Application Opened"',
```